app/assets/stylesheets/cart.css.scss

```diff
@@ -26,3 +26,17 @@
 #checkout {
     background: green;
 }
+
+#delete-link {
+  font-family: "Open Sans";
+  a:link {color: #000000; text-decoration: underline; }
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.